app/controllers/faq_controller.rb

```diff
@@ -6,7 +6,7 @@ class FaqController < ApplicationController
   DEFAULT_TOPIC = 'top'
 
   def index
-    @query = params[:search]
+    @query = (params[:search] == "") ? nil : params[:search]
```

Comment:
I think this might be easier to understand like `@query = params[:search] if params[:search].present?`

Reply:
Done @ragesoss